test_webhook.py

```
@@ -0,0 +1,291 @@
+        """Clean up test environment."""
+        # Restore the original CONTROL_API_TOKEN value
+        if self.original_control_api_token is not None:
+            os.environ["CONTROL_API_TOKEN"] = self.original_control_api_token
+        elif "CONTROL_API_TOKEN" in os.environ:
+            del os.environ["CONTROL_API_TOKEN"]
+    
+    def test_control_start_no_auth(self):
+        """Test that control start endpoint requires authentication."""
+        payload = {"story_id": "test_story", "date": "2025-01-01"}
+        response = client.post("/control/record/start", json=payload)
+        assert response.status_code == 401
```

Ruff (0.12.2)

Use of `assert` detected

(S101)